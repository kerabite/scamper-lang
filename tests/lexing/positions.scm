(define x 10)

(define y "hello")

(define f
  (lambda (y) (+ x y)))