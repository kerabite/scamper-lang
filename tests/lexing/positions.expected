(: (1, 1) => (1, 1)
define: (1, 2) => (1, 7)
x: (1, 9) => (1, 9)
10: (1, 11) => (1, 12)
): (1, 13) => (1, 13)
(: (3, 1) => (3, 1)
define: (3, 2) => (3, 7)
y: (3, 9) => (3, 9)
"hello": (3, 11) => (3, 17)
): (3, 18) => (3, 18)
(: (5, 1) => (5, 1)
define: (5, 2) => (5, 7)
f: (5, 9) => (5, 9)
(: (6, 3) => (6, 3)
lambda: (6, 4) => (6, 9)
(: (6, 11) => (6, 11)
y: (6, 12) => (6, 12)
): (6, 13) => (6, 13)
(: (6, 15) => (6, 15)
+: (6, 16) => (6, 16)
x: (6, 18) => (6, 18)
y: (6, 20) => (6, 20)
): (6, 21) => (6, 21)
): (6, 22) => (6, 22)
): (6, 23) => (6, 23)
